packages/react-components/src/components/ImageGallery.test.tsx

```diff
@@ -24,92 +24,72 @@ describe('ImageGallery default layout tests', () => {
-    title: 'title2',
-    titleIcon: <div data-testid={titleIconTestId2}></div>
-  };
-  /* @conditional-compile-remove(image-gallery) */
-  const renderImageGalleryComponent = (
-    images?: Array<ImageGalleryImageProps>,
-    startIndex?: number,
+
+  /* @conditional-compile-remove(image-overlay) */
+  const renderImageOverlayComponent = (
+    imageSrc?: string,
+    title?: string,
+    titleIcon?: JSX.Element,
     onDismiss?: () => void,
-    onImageDownloadButtonClicked?: () => void,
+    onDownloadButtonClicked?: () => void,
     onError?: () => void
   ): HTMLElement => {
-    const imagesArray = images || [imageInfo];
     const { container } = render(
-      <ImageGallery
-        images={imagesArray}
-        startIndex={startIndex}
+      <ImageOverlay
+        imageSrc={imageSrc || imageInfo.imageSrc}
+        title={title || imageInfo.title}
+        titleIcon={titleIcon || imageInfo.titleIcon}
         onDismiss={onDismiss || jest.fn()}
-        onImageDownloadButtonClicked={onImageDownloadButtonClicked || jest.fn()}
+        onDownloadButtonClicked={onDownloadButtonClicked || jest.fn()}
         onError={onError || jest.fn()}
         isOpen={true}
       />
     );
     return container;
   };
-  /* @conditional-compile-remove(image-gallery) */
-  test('Show image gallery with required props', async () => {
-    renderImageGalleryComponent();
-    const image: HTMLImageElement = await screen.findByRole('img', { name: 'image-gallery-main-image' });
+  /* @conditional-compile-remove(image-overlay) */
+  test('Show image overlay with required props', async () => {
+    renderImageOverlayComponent();
+    const image: HTMLImageElement = await screen.findByRole('img', { name: 'image-overlay-main-image' });
     const title: HTMLElement = await screen.findByText(imageInfo.title);
     const titleIcon: HTMLElement = await screen.findByTestId(titleIconTestId1);
-    expect(image.src).toContain(imageInfo.imageUrl);
+    expect(image.src).toContain(imageInfo.imageSrc);
     expect(image.alt).toBe(imageInfo.altText);
 
     expect(title).toBeTruthy();
     expect(titleIcon).toBeTruthy();
   });
-  /* @conditional-compile-remove(image-gallery) */
-  test('Show the correct image from images base on the startIndex', async () => {
-    renderImageGalleryComponent([imageInfo, imageInfo2], 1);
-    const image: HTMLImageElement = await screen.findByRole('img', { name: 'image-gallery-main-image' });
-    const title: HTMLElement = await screen.findByText(imageInfo2.title);
-    const titleIcon: HTMLElement = await screen.findByTestId(titleIconTestId2);
-    expect(image.src).toContain(imageInfo2.imageUrl);
-    expect(image.alt).toBe(imageInfo2.altText);
-
-    expect(title).toBeTruthy();
-    expect(titleIcon).toBeTruthy();
-  });
 
-  /* @conditional-compile-remove(image-gallery) */
+  /* @conditional-compile-remove(image-overlay) */
   test('It should call the onDismiss handler when the close icon is clicked', async () => {
```

Comment:
Do we still have the functionality to close the overlay when users taps outside of the image?

Reply:
> Do we still have the functionality to close the overlay when users taps outside of the image?

Yes this functionality remain unchanged